### PR TITLE
Issue 244

### PR DESCRIPTION
**Pull request by rpiazza**
elevator can create content that uses the new extension-definitions.

The extension-definitions are in extension_definitions.py.  They are just stand-ins.  There is no json schema behind them

### CODE REVIEW

File: stix2elevator/extension_definitions.py [hunk 1]
```diff
@@ -0,0 +1,270 @@
+MITRE_COPYRIGHT_STATEMENT_DATA_MARKING = {
+    "created": "2020-12-22T00:00:00.000000Z",
+    "definition": {
+        "statement": "Copyright 2013 FireEye, Inc."
```
Comment (emmanvg) [inline stix2elevator/extension_definitions.py:4]: Update statement string... (MITRE Corporation)

File: stix2elevator/extension_definitions.py [hunk 2]
```diff
@@ -0,0 +1,270 @@
...
+        "spec_version": "2.1",
+        "name": "Extension to support STIX 1.x archive-file",
+        "description": "This schema adds two properties to a STIX object",
+        "created": "2020-12-22T00:00:00.000000Z",
+        "modified": "2020-12-22T00:00:00.000000Z",
+        "created_by_ref": MITRE_IDENTITY_OBJECT["id"],
+        "object_marking_refs": [
+            MITRE_COPYRIGHT_STATEMENT_DATA_MARKING["id"]
+        ],
+        "schema": "https://https://github.com/mitre/cti/extension-definitions/stix-1-archive-file/",
```
Comment (emmanvg) [inline stix2elevator/extension_definitions.py:37]: All `schema` property URLs have duplicate `https://https://`

File: stix2elevator/extension_definitions.py [hunk 3]
```diff
@@ -0,0 +1,270 @@
...
+        "version": "1.0.0",
+        "extension_types": ["property-extension"]
+    },
+    "attack-pattern": {
+        "id": "extension-definition--d83fce45-ef58-4c6c-a3f4-1fbc32e98c6e",
```
Comment (emmanvg) [inline stix2elevator/extension_definitions.py:42]: Calculate new UUID for these: `attack-pattern` and `campaign` clash

File: stix2elevator/extension_definitions.py [hunk 4]
```diff
@@ -0,0 +1,270 @@
...
+        "created_by_ref": MITRE_IDENTITY_OBJECT["id"],
+        "object_marking_refs": [
+            MITRE_COPYRIGHT_STATEMENT_DATA_MARKING["id"]
+        ],
+        "schema": "https://https://github.com/mitre/cti/extension-definitions/stix-1-campaign/",
+        "version": "1.0.0",
+        "extension_types": ["property-extension"]
+    },
+    "course-of-action": {
+        "id": "extension-definition--a46b18de-0b41-4a95-9d2d-67a360f2d859",
```
Comment (emmanvg) [inline stix2elevator/extension_definitions.py:74]: Calculate new UUID for these: `course-of-action` and `email-message` clash

File: stix2elevator/extension_definitions.py [hunk 5]
```diff
@@ -0,0 +1,270 @@
...
+        "created_by_ref": MITRE_IDENTITY_OBJECT["id"],
+        "object_marking_refs": [
+            MITRE_COPYRIGHT_STATEMENT_DATA_MARKING["id"]
+        ],
+        "schema": "https://https://github.com/mitre/cti/extension-definitions/stix-1-email-message/",
+        "version": "1.0.0",
+        "extension_types": ["property-extension"]
+    },
+    "icmp-header": {
+        "id": "extension-definition--6f6973cb-70be-40cb-b1ac-15ed78cebe56",
```
Comment (emmanvg) [inline stix2elevator/extension_definitions.py:106]: Calculate new UUID for these: `icmp-header`, `malware` and `network-socket` clash

File: idioms-json-2.0-custom/ioc-indicator.json [hunk 6]
```diff
@@ -10,6 +10,7 @@
             "modified": "2020-02-24T16:07:27.201Z",
             "name": "Zeus",
             "type": "malware",
+            "x_elevator_other_names": "twexts,sdra64,ntos",
```
Comment (emmanvg) [inline idioms-json-2.0-custom/ioc-indicator.json:13]: Would it be better to represent this as a list of strings? instead of the comma-separated string...

Comment (rpiazza) [reply idioms-json-2.0-custom/ioc-indicator.json:13]: I think this was fixed in the next PR

**Comment by emmanvg** [comment]
PR #247 already contains changes and suggestions from this request

**Pull request closed without merging**